package.json

```diff
@@ -44,5 +44,8 @@
   "bugs": {
     "url": "https://github.com/mongodb-js/mongodb-core/issues"
   },
-  "homepage": "https://github.com/mongodb-js/mongodb-core"
+  "homepage": "https://github.com/mongodb-js/mongodb-core",
+  "optionalDependencies": {
```

Comment:
should we switch all of our `peerOptionalDependencies` to just be `optionalDependencies` 🤔

Reply:
I just went and saw a number of issues users were having with `require_optional` (e.g. [this](https://github.com/christkv/require_optional/issues/3)), maybe we should discuss removing it in favor of just `optionalDependencies`